cli/command/cli.go

```diff
@@ -1,8 +1,8 @@
 package command
 
 import (
-	"errors"
 	"fmt"
+	"github.com/pkg/errors"
```

Comment:
We should fix the weird import positioning before merging this. It can be done manually if necessary.